pco/src/data_types/floats.rs

```diff
@@ -177,6 +197,9 @@ macro_rules! impl_float_number {
           Mode::FloatMult(base_latent) => {
             Self::from_latent_ordered(base_latent).is_finite_and_normal()
           }
+          Mode::FloatDecomp(k_latent) => {
```

Comment:
`k` isn't a latent, right? It's just a `Bitlen`? And since it's unsigned, it can't be < 0